app/controllers/account/tasks_controller.rb

```diff
@@ -1,56 +1,67 @@
 class Account::TasksController < Account::AccountController
-  before_action :set_project
-  before_action :set_task, only: [:show, :edit, :update, :destroy]
+  before_action :parent
```

Comment:
make it more verbose

Reply:
Remove this before action. Just call "parent" where you need it. @workspace = parent. In action itself